pipeline_steps/influx.groovy

```diff
@@ -0,0 +1,67 @@
+def setup(){
+  instance_name = "INFLUX"
+  pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
+  common.override_inventory()
+  try{
+    common.conditionalStage(
+      stage_name: "Influx",
+      stage:{
+        withCredentials([
+          file(
+            credentialsId: 'id_rsa_cloud10_jenkins_file',
+            variable: 'jenkins_ssh_privkey'
+          ),
+        ]){
+          dir('/tmp/'){
+            unstash "pubcloud_inventory"
```

Comment:
Whoops, you are correct, this stash is created in pubcloud.groovy and will exist. Apologies. You still don't need to use stash, but the alternative is copying out of workspace/rpc-gating/playbooks and I don't think thats any neater.